modelscan/skip.py

```diff
@@ -1,17 +1,18 @@
 import logging
 from enum import Enum
 
+from modelscan.settings import Property
 
 logger = logging.getLogger("modelscan")
 
 
-class SkipCategories(Enum):
-    SCAN_NOT_SUPPORTED = 1
-    BAD_ZIP = 2
-    MODEL_CONFIG = 3
-    H5_DATA = 4
-    NOT_IMPLEMENTED = 5
-    MAGIC_NUMBER = 6
+class SkipCategories:
+    SCAN_NOT_SUPPORTED = Property("SCAN_NOT_SUPPORTED", 1)
+    BAD_ZIP = Property("BAD_ZIP", 2)
+    MODEL_CONFIG = Property("MODEL_CONFIG", 3)
+    H5_DATA = Property("H5_DATA", 4)
+    NOT_IMPLEMENTED = Property("NOT_IMPLEMENTED", 5)
+    MAGIC_NUMBER = Property("MAGIC_NUMBER", 6)
```

Comment:
Would IntEnum be better here?

<img width="443" alt="Screenshot 2024-05-23 at 5 03 02 PM" src="https://github.com/protectai/modelscan/assets/8380706/c360882b-caf8-4bdc-b9c9-fa89fd360862">

Reply:
same problem with not being extendable